cmd/autoowners/main.go

```diff
@@ -48,12 +51,13 @@ func (orgRepo orgRepo) repoString() string {
 	return fmt.Sprintf("%s/%s", orgRepo.Organization, orgRepo.Repository)
 }
 
-func loadRepos(dir string, blacklist sets.String) ([]orgRepo, error) {
+func loadRepos(configRootDir string, blacklist sets.String, sourceSubDirs []string, githubOrg string, githubRepo string) ([]orgRepo, error) {
 	var orgRepos []orgRepo
-	operatorRoot := filepath.Join(dir, "ci-operator")
-	jobs := filepath.Join(operatorRoot, "jobs")
-	config := filepath.Join(operatorRoot, "config")
-	templates := filepath.Join(operatorRoot, "templates")
+	configSubDirectories := make([]string, 0, len(sourceSubDirs))
+	for _, sourceSubDir := range sourceSubDirs {
+		configSubDirectories = append(configSubDirectories, filepath.Join(configRootDir, sourceSubDir))
+	}
+	jobs := configSubDirectories[0]
```

Comment:
This looks fishy to me. It basically means that the first item in the list has some special meaning, but this is not documented anywhere and even if it were, I still think it's a smell.

Reply:
Logically it has a meaning, as 
* if it's not present, an error is returned
* if the org and repo dirs are not present in this directory, it is skipped
Maybe we should just make that clear in the documentation that the first directory name in the `configSubDirs` list must contain all the org/repo dirs, or else it will be skipped